bigbluebutton-client/src/org/bigbluebutton/modules/screenshare/managers/WebRTCDeskshareManager.as

```diff
@@ -166,7 +167,7 @@ package org.bigbluebutton.modules.screenshare.managers
 			 sends out fall back to java command
 		*/
 		private function cannotUseWebRTC (message:String):void {
-			LOGGER.debug("WebRTCDeskshareManager::handleStartSharingEvent - falling back to java");
+			JSLog.warn("Cannot use WebRTC Screensharing: ", message);
```

Comment:
Added.

I didn't really feel it was necessary because JSLog would allow both developers/users to see it easily so it seemed kind of like a 2 for 1 putting it in JSLog. But I guess having all messages logged through the logger is beneficial so easily added it